### PR TITLE
Add GAT 2023 blog post

### PR DESCRIPTION
**Pull request by sanjaysrikakulam**
@sebastian-schaaf, this is a first draft. Please share your comments and suggestions, if any. Please also correct the funding section.

### CODE REVIEW

File: content/news/2023-09-27-galaxy-admin-training/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+hide_tease: false
+subsites: [all-eu]
+main_subsite: eu
+---
+
+Galaxy Europe conducted the [Galaxy Admin Training (GAT) 2023](https://galaxyproject.org/events/2023-admin-training/) in Ghent Belgium this year from April 17th till 21st and invited experts and senior admins with several years of experience in maintaining Galaxy to share the ins and outs of setting up an own production quality Galaxy server and various other services. The training was attended by 16 participants from different countries and backgrounds.
+
+The training covered the basic and advanced topic needed to know to setup and maintain a production, high-performance and multi-user Galaxy instance. During the one week intensive hands-on training session participants were introduced to Ansible a configuration management system, which is used to automate the installation and configuration of Galaxy and its dependencies. The training also covered the installation of various other services such as tusd, CVMFS, containers, tool management, reference datasets, meta-schedulers such as Total Perspective Vortex (TPV), job schedulers, pulsar, celery among others.
```
Comment (kysrpex) [inline content/news/2023-09-27-galaxy-admin-training/index.md:12]:
```suggestion
The training covered the basic and advanced topics needed to know to setup and maintain a production, high-performance and multi-user Galaxy instance. During the one week intensive hands-on training session participants were introduced to Ansible, a configuration management system which is used to automate the installation and configuration of Galaxy and its dependencies. The training also covered the installation of various other services such as tusd, CVMFS, containers, tool management, reference datasets, meta-schedulers such as Total Perspective Vortex (TPV), job schedulers, pulsar, celery among others.
```

Typo hints. Thanks for writing the blog post.

File: content/news/2023-09-27-galaxy-admin-training/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+During the training, various elements, strategies, debugging techniques and best practices were discussed and shared. The training was a great success and the participants were very happy with the training and the knowledge they gained. Especially with the training material, which is available online and can be used as a reference for future use. The training material is available at [https://training.galaxyproject.org/training-material/topics/admin/](https://training.galaxyproject.org/training-material/topics/admin/).
+
+# Funding
+
+We would like to thank the [EOSC-LIFE](https://www.eosc-life.eu/) and [EOSC EuroScienceGateway](https://galaxyproject.org/projects/esg/) projects for their support and funding for this training. EOSC-Life has received funding from the European Union’s Horizon 2020 programme under grant agreement number 824087. EuroScienceGateway has received funding from Horizon Europe programme of the EU with the number 101057388.
```
Comment (bgruening) [inline content/news/2023-09-27-galaxy-admin-training/index.md:18]: Please add ELIXIR as well.

**Comment by bgruening** [comment]
Can we please add the schedule and link the training materials that we did.

**Comment by kysrpex** [comment]
> Can we please add the schedule and link the training materials that we did.

For the link to the training materials: isn't it already in the text? https://training.galaxyproject.org/training-material/topics/admin/

**Comment by kysrpex** [comment]
> Can we please add the schedule and link the training materials that we did.

Regarding the schedule, [this one](https://github.com/orgs/galaxyproject/projects/28/views/1?filterQuery=status%3AMonday%2CTuesday%2CWednesday%2CThursday%2CFriday+)? There exists also an [Event page](https://galaxyproject.org/events/2023-admin-training).

**Comment by kysrpex** [comment]
Just to facilitate things I am also linking to the [GitHub repo](https://github.com/galaxyproject/admin-training).

File: content/news/2023-09-27-galaxy-admin-training/index.md (hunk 2 above)
Comment (sanjaysrikakulam) [reply content/news/2023-09-27-galaxy-admin-training/index.md:18]: Is there a grant number that needs to be mentioned?

**Comment by sanjaysrikakulam** [comment]
My bad, I thought the event page already included the schedule and the github project. I will shortly push a commit with the required changes.

**Comment by kysrpex** [comment]
> My bad, I thought the event page already included the schedule and the github project. I will shortly push a commit with the required changes.

The event page has a link to the schedule at least, but it is kind of broken though. It has the wrong filters, so you do not see the schedule for all days.

**Comment by kysrpex** [comment]
@bgruening What Sanjay is saying is true, he is already linking to the event page that links to the schedule (and editing the default filters for the schedule would do it).

**Comment by bgruening** [comment]
Fixing the event page makes sense. Copy and pasting the schedule here is also ok I think. This is for reporting to the funders. It needs to look great without clicking through subpages.

**Comment by sanjaysrikakulam** [comment]
I am adding them exclusively to the report. I am just struggling to find the ELIXIR official logo URL.

**Comment by sanjaysrikakulam** [comment]
Added the requested changes.

**Pull request merged**